lib/ui/page/call/component/mobile.dart

```diff
@@ -140,784 +108,726 @@ Widget mobileCall(CallController c, BuildContext context) {
+                        ),
+                      ),
+                      child: ScreenButton(c).build(),
+                    )),
+                  HorizontalPadding(
+                      child: Description(
+                    description: AnimatedOpacity(
                       opacity: c.isPanelOpen.value ? 1 : 0,
                       duration: 200.milliseconds,
                       child: Text(
-                        c.screenShareState.value == LocalTrackState.enabled ||
-                                c.screenShareState.value ==
-                                    LocalTrackState.enabling
-                            ? 'btn_call_screen_off_desc'.l10n
-                            : 'btn_call_screen_on_desc'.l10n,
+                        c.audioState.value == LocalTrackState.enabled ||
+                                c.audioState.value == LocalTrackState.enabling
+                            ? 'btn_call_audio_off_desc'.l10n
+                            : 'btn_call_audio_on_desc'.l10n,
                       ),
                     ),
+                    child: AudioButton(c).build(),
                   )),
-                padding(withDescription(
-                  AudioButton(c).build(),
-                  AnimatedOpacity(
-                    opacity: c.isPanelOpen.value ? 1 : 0,
-                    duration: 200.milliseconds,
-                    child: Text(
-                      c.audioState.value == LocalTrackState.enabled ||
-                              c.audioState.value == LocalTrackState.enabling
-                          ? 'btn_call_audio_off_desc'.l10n
-                          : 'btn_call_audio_on_desc'.l10n,
+                  HorizontalPadding(
+                      child: Description(
+                    description: AnimatedOpacity(
+                      opacity: c.isPanelOpen.value ? 1 : 0,
+                      duration: 200.milliseconds,
+                      child: Text(
+                        c.videoState.value == LocalTrackState.enabled ||
+                                c.videoState.value == LocalTrackState.enabling
+                            ? 'btn_call_video_off_desc'.l10n
+                            : 'btn_call_video_on_desc'.l10n,
+                      ),
                     ),
-                  ),
-                )),
-                padding(withDescription(
-                  VideoButton(c).build(),
-                  AnimatedOpacity(
-                    opacity: c.isPanelOpen.value ? 1 : 0,
-                    duration: 200.milliseconds,
-                    child: Text(
-                      c.videoState.value == LocalTrackState.enabled ||
-                              c.videoState.value == LocalTrackState.enabling
-                          ? 'btn_call_video_off_desc'.l10n
-                          : 'btn_call_video_on_desc'.l10n,
+                    child: VideoButton(c).build(),
+                  )),
```

Comment:
Не забываем про форматирование.

```suggestion
                  ),),
```